ui/webui/resources/polymer_overriding.js

```diff
@@ -27,6 +29,16 @@ function addBraveBehaviors(moduleName, component) {
   }
 }
 
+function addBraveProperties(moduleName, component) {
+  if (allPropertiesMap[moduleName]) {
+    component.properties = component.properties || {}
+    Object.assign(component.properties, allPropertiesMap[moduleName])
```

Comment:
I think this would be cleaner to understand with 
```js
component.properties = {
  ...component.properties,
  ...allPropertiesMap[moduleName]
}
```